examples/webgl_lights_rectarealight.html

```diff
@@ -56,12 +56,12 @@
 
 				}
 
-				if ( renderer.capabilities.isWebGL2 === false && ! renderer.extensions.get( 'OES_texture_float_linear' ) ) {
+				// if ( renderer.capabilities.isWebGL2 === false && ! renderer.extensions.get( 'OES_texture_float_linear' ) ) {
```

Comment:
Since this PR improves the platform support of `RectAreaLight` by using `OES_texture_half_float`, the capability checks seems not necessary anymore. I would also remove the comments.

Reply:
Are there any platforms that Three.js wishes to support for which this expression evaluates to false?

```
(renderer.capabilities.isWebGL2 || 
renderer.extensions.get( 'OES_texture_float_linear' ) ||
renderer.extensions.get( 'OES_texture_half_float_linear'))
``

Reply:
No, I don't think so.

Reply:
No, I don't think so.

Reply:
OK. I've removed those checks from the example. Meanwhile `RectAreaLightUniformsLib.init()` throws an error if it doesn't find WebGL2 or one of those extensions.